pandas/core/arrays/interval.py

```diff
@@ -1217,3 +1279,55 @@ def maybe_convert_platform_interval(values):
         values = np.asarray(values)
 
     return maybe_convert_platform(values)
+
+
+if _PYARROW_INSTALLED and LooseVersion(pyarrow.__version__) >= LooseVersion("0.15"):
```

Comment:
__PYARROW_INSTALLED needs to incorporate the version check

Reply:
I moved the version check into the separate file (and made it a variable), but kept it separate from the import check as different functionalities might need a different pyarrow version